dev.py

```diff
@@ -46,6 +46,26 @@ def cmd_all_npm_build(args):
         frontend_dir = next(project_dir.glob("*/frontend/"))
         run_verbose(["npm", "run", "build"], cwd=str(frontend_dir))
 
+
+def cmd_all_install_python_deps(args):
```

Comment:
Comments can be added to these commands to make them visible from "./dev.py --help"

Reply:
done